docs/cli.md

```diff
@@ -0,0 +1,114 @@
+# Ara Module Template Command Line Guide
```

Comment:
Point to cli-docs-generator script or hold off until it supports more formats? :)

Reply:
mmm... let's wait til it is used by at least one of aid, ann, act, or afs